Google.Cloud.EntityFrameworkCore.Spanner.Tests/EntityFrameworkMockServerTests.cs

```diff
@@ -160,14 +163,23 @@ public async Task InsertSinger_SelectsFullName()
                 }
             );
             Assert.Collection(
-                _fixture.SpannerMock.Requests.Where(request => request is V1.ExecuteSqlRequest).Select(request => (V1.ExecuteSqlRequest)request),
+                _fixture.SpannerMock.Requests.Where(request => request is ExecuteSqlRequest).Select(request => (ExecuteSqlRequest)request),
                 request =>
                 {
                     Assert.Equal(selectFullNameSql, request.Sql);
-                    Assert.NotNull(request.Transaction?.Id);
+                    Assert.Null(request.Transaction?.Id);
```

Comment:
The propagation of computed columns has been changed to run __outside__ of the transaction when changes are saved in an implicit transaction. This is needed in order to support result propagation of changes that are saved using mutations, as mutations are only visible after a `Commit`, but it also has the additional advantage that it is more efficient if the user chooses to use DML with implicit transactions.